### ISSUE TITLE
Vi har også 'havenes akustikk' og 'undervannsakustikk'

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#271]
Nørvendig med begge? Foreslår samanslåing

**Comment by danmichaelo** [comment]
Enig. Som fysiker tenker jeg at vann er vann :D Også kan en jo bruke emneord som `Oseanografi` for å avgrense til hav.

Forresten, vi har visst `Hydroakustikk` også som eget emne. Jeg tror vi kan slå den også inn i `Undervannsakustikk` @mittinatten

**Comment by mittinatten** [comment]
Jeg mener `Hydroakustikk` burde være foretrukket, selv om ordet er dobbelt så gresk somundervannsakustikk. På samme måte som vi bruker fluiddyamikk og ikke vanndynamikk eller væskedynamikk, etc.

**Comment by mittinatten** [comment]
For å være korrekt skulle jeg ha sagt hydrodynamikk i stedet for vanndynamikk, og fluiddynamikk i stedet for væskedynamikk. :)

**Comment by Hypsibius** [comment]
Om det då ikkje var 'Hagenes akustikk' det var snakk om over...

**Comment by mittinatten** [comment]
Ja, det er nok så viktig, hvis vi ikke har belegg kanskje du kan ta tak i å skrive boken?

**Comment by danmichaelo** [comment]
(Den bør handle om hvilke hekker/trær/busker som best demper/reflekterer lyd fra naboen :house_with_garden:)

Men ja... jeg var litt i tvil om foretrukket term selv. Nå omfatter jo "fluid" mer enn "væske", og situasjonen er annerledes enn for hydrodynamikk siden både undervannsakustikk og hydrodynamikk er i utstrakt bruk..

Morsomt nok har GND landet på `Hydroakustik BF Unterwasserakustik` (http://d-nb.info/gnd/4259255-0) , mens LCSH har `Underwater acoustics BF Hydroacoustics` (http://id.loc.gov/authorities/subjects/sh85139609)

(Og på engelsk Wikipedia er det noen som har forsøkt å skille de to termene, men det virket ikke helt overbevisende :))

Neeei, jeg vet ikke, så vi kan godt ta `Hydroakustikk` som foretrukket :)

```
Hydroakustikk
BF Undervannsakustikk
BF Marin akustikk
EN Hydroacoustics
EN Underwater acoustics
```

… men så oppdager jeg at vi har `Akustisk oseanografi` også :D (da var vi over i `Havenes akustikk` igjen) En gruppe ved NTNU [beskriver](http://www.iet.ntnu.no/groups/akustikk/hydrogeo/) dette som «anvendelse av marin akustikk (eller undervannsakustikk)». Vi kunne nok beholdt det som eget begrep, men jeg ser litt for meg at det kan bli vanskelig å skille i praksis? Tar vi sjansen på å smashe inn `BF Akustisk oseanografi EN Ocean acoustics EN Acoustical oceanography` i `Hydroakustikk` også? Vi kan jo legge på en liten huskelappnote om at `Oseanografi` også bør legges til for dokumenter om `Akustisk oseanografi`, slik at det går an å avgrense.

**Comment by mittinatten** [comment]
Har @mzyg noen mening her?

**Comment by mzyg** [comment]
BF Akustisk oseanografi EN Ocean acoustics EN Acoustical oceanography bor i hvert fall være med. Jobbet tidligere en del sammen med folk som brukte bare disse termene og ikke undervannsakustikk/marin akustikk.

**Comment by mittinatten** [comment]
Så mener du vi bør ha to separate termer eller holder det å legge alt inn under hydroakustikk?

**Comment by mzyg** [comment]
Det holder å legge det inn under hydroakustikk.

**Comment by danmichaelo** [comment]
Supert! Da har vi
```
Hydroakustikk
BF Undervannsakustikk
BF Marin akustikk
BF Akustisk oseanografi
EN Hydroacoustics
EN Underwater acoustics
EN Ocean acoustics
EN Acoustical oceanography
```

**Comment by Hypsibius** [comment]
Her går det unna!

**Comment by bibliomari** [comment]
Gjort.

**Comment by Hypsibius** [comment]
Gjer meg litt vanskeleg her, men vi bør gjerne ha ein tilsvarande gjennomgang av 'havenes optikk'. som eg heller ikkje synest er ein glimrande term?

**Comment by BioHeidi** [comment]
Drar opp igjen denne for å se nærmere på ekkolodd, ekkolokalisering, bioakustikk, akustiske signaler, sonarer, og mulige sammenblandinger og forvirringer ute og går der.
